healthcheck/backends/__init__.py

```diff
@@ -18,12 +18,21 @@ def get_value(key):
     return value
 
 
+def get_value_or_default(key, default):
```

Comment:
Same here for os.environ. Since it is a dict, it could be rewrite as:

```
def get_value_or_default(key, default):
   return os.environ.get(key) or default
```